downstream/modules/platform/proc-create-a-user.adoc

```diff
@@ -0,0 +1,52 @@
+[id="proc-create-a-user_{context}"]
+
+= Creating a User
+
+This procedure creates a user that can be used to log in to {HubName}.
+As the `hubadmin` role is assigned, this user has Super Administration privileges when logged in.
+
+Use the following procedure to create a Keycloak User.
+
+.Procedure
+
+. Navigate to menu:Operator[Installed Operators].
+. Select the Red Hat Single Sign-On Operator project.
+. Select the *Keycloak Realm* tab and click btn:[Create Keycloak User].
+. On the Keycloak User form, select btn:[YAML view].
+Replace the default YAML file with the following:
++
+[options="nowrap" subs="+quotes"]
+----
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakUser
+metadata:
+  name: hubadmin-user
+  labels:
+    app: sso
+    realm: ansible-automation-platform
+  namespace: rh-sso
+spec:
+  realmSelector:
+    matchLabels:
+      app: sso
+      realm: ansible-automation-platform
+  user:
+    username: hub_admin
+    firstName: Hub
+    lastName: Admin
+    email: hub_admin@example.com
+    enabled: true
+    emailVerified: false
+    credentials:
+      - type: password
+        value: ch8ngeme
+    clientRoles:
+      automation-hub:
+        - hubadmin
+----
+
+. Click btn:[Create] and wait for the process to complete.
+
+When a user is created, the Operator creates a Secret containing both the username and password using the following naming pattern: `credential-<realm name>-<username>-<namespace>`.
+In this example the credential is called `credential-ansible-automation-platform-hub-admin-rh-sso`.
+When a user is created the operator does not update the user's password and password changes will not be reflected in the Secret.
```

Comment:
I'm not really sure what this last sentence means. It's also a run on sentence (two independent clauses joined by a conjunction).